code/unclassified/src/average/Average.java

```diff
@@ -0,0 +1,37 @@
+import java.util.*;
+// Part of Cosmos by OpenGenus Foundation
+class Average
+{
+    /**
+    * @param arr array of integers to get the average of.
+    * @param n number of elements.
+    * @return the average calculated as sum(arr)/n
+    */
+    static double avrg(Vector<Integer> arr)
+    {
+        if (arr.isEmpty()) return 0.0;
+        double sum = 0;
+        double avg = 0.0;
+        final int SIZE = arr.size();
+        for(int i = 0;i < SIZE; i++)
```

Comment:
If you use a Java `foreach` loop, then you can get rid of the variable above and the code is somewhat clearer.